### PR TITLE
Docker image

### PR DESCRIPTION
**Pull request by atamariya**
Scripts to create a docker image for ums. Mount following volumes and ports to run the image:

    Media folder
    VOLUME /media

    Profile folder containing UMS.conf
    VOLUME /profile

EXPOSE 9001 5002 1044

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+WORKDIR /usr/src/ums
+
+VOLUME /media
+
+VOLUME /profile
+
+EXPOSE 9001 5002 1044
+
+CMD java -DUMS_PROFILE=/profile -Dfile.encoding=UTF-8 -Djava.net.preferIPv4Stack=true -Djna.nosys=true -cp ums.jar net.pms.PMS
```
Comment (SubJunk) [inline docker/Dockerfile]: Does it need memory instructions like 1280mb?

Comment (atamariya) [reply docker/Dockerfile]: Added JVM_OPTS environment variable for runtime memory setting. However, memory allotted to container will also play an important role. https://github.com/akashche/docs/blob/4325be5ccb52721bfa26521beaee40e607c7b7af/openjdk/content.md

Comment (SubJunk) [reply docker/Dockerfile]: Ok great, thanks. That looks like a fine default. We have historically used 768mb for 32-bit and 1280mb for 64-bit systems. I don't know enough about docker to properly review this PR but I can do some reading about it, or do you think it is just safe to merge now?

Comment (atamariya) [reply docker/Dockerfile]: It's safe to merge.

**Comment by atamariya** [comment]
@SubJunk You should publish a docker image at https://hub.docker.com/ and also add an announcement in forum for better visibility. 
```
cd docker
ant
docker tag ums $DOCKER_ID/ums
docker push $DOCKER_ID/ums
```

Users can pull the image using:
```
docker pull $DOCKER_ID/ums
docker run -it ums
```



```

**Comment by SubJunk** [comment]
Ok, will do

**Comment by atamariya** [comment]
@SubJunk created PR #1599

**Comment by SubJunk** [comment]
@atamariya ok I have verified that the binaries are working now, they were in the wrong place in the image. I've also added instructions to the UMS release script I use so I can't forget to update it now.

**Pull request merged**